app/src/main/java/com/amazonaws/services/chime/sdkdemo/fragment/MeetingFragment.kt

```diff
@@ -268,7 +268,7 @@ class MeetingFragment : Fragment(),
         } ?: AudioMode.Stereo48K
         val audioVideoConfig = AudioVideoConfiguration(audioMode = audioMode)
         // Update the Mic & Speaker states
-        updateLocalAttendeeAudioState(audioEnabled = audioVideoConfig.audioMode != AudioMode.NoAudio)
+        updateLocalAttendeeAudioState(audioEnabled = true)
```

Comment:
We may remove this method if we don't want to give any hint about NoAudio support

Reply:
Removed